LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+Font license info
+
```

Comment:
Jak już zamieszczamy licencje bibliotek zewnętrzych, zróbmy to w osobnym katalogu i ponazywajmy te pliki odpowiednio. A najlepszym rozwiązaniem jest utworzenie pliku `README.md` i w nim opisanie jakie biblioteki zewnętrzne zostały wykorzystane w projekcie i na jakiej licencji.